README.md

```diff
@@ -248,6 +249,17 @@ Linter-Specific Option | Description
 `rubocop_config`       | A valid rubocop configuration hash. Mandatory when this cop is enabled. See [rubocop's manual entry on Configuration](http://rubocop.readthedocs.io/en/latest/configuration/)
 `only`                 | Only run cops listed in this array instead of all cops.
 
+### RequireInputAutocomplete
+This linter prevents the usage of certain types of HTML `<input>` without an `autocomplete` argument: `color`, `date`, `datetime-local`, `email`, `hidden`, `month`, `number`, `password`, `range`, `search`, `tel`, `text`, `time`, `url`, or `week`.
+The HTML autocomplete helps users to complete filling in forms by using data stored in the browser. This is particularly useful for people with **motor disabilities** or **cognitive impairment** who may have difficulties filling out forms online.
+
+```
+Bad ❌
+<input type="email" ...>
+Good ✅
+<input type="email" autocomplete="email" ...>
```

Comment:
Can we also add an example of when you don't want autocomplete?
`<input type="email" autocomplete="nope" ...>`